apps/sdk-explorer/src/components/Hooks.tsx

```diff
@@ -1,57 +1,34 @@
 import {Badge, Box, Inline} from '@sanity/ui'
 import {type JSX} from 'react'
 
-const baseUrl = 'https://sdk-docs.sanity.dev/functions/React_SDK.exports.'
+const baseUrl = 'https://reference.sanity.dev/_sanity/sdk-react/exports/'
```

Comment:
Shouldn't this be io?
```suggestion
const baseUrl = 'https://reference.sanity.io/_sanity/sdk-react/exports/'
```